extensions/binding/org.eclipse.smarthome.binding.dmx/src/main/java/org/eclipse/smarthome/binding/dmx/internal/multiverse/DmxChannel.java

```diff
@@ -253,8 +254,10 @@ public synchronized Integer getNewHiResValue(long calculationTime) {
         if (hasRunningActions()) {
             BaseAction action = actions.get(0);
             value = action.getNewValue(this, calculationTime);
-            if (action.isCompleted()) {
+            if (ActionState.completed.equals(action.getState())) {
```

Comment:
since `ActionState` are enum values this should be `if (action.getState() == ActionState.COMPLETED) {`. please check other comparison, too.